docs/source/conf.py

```diff
@@ -140,12 +140,12 @@
     'matplotlib': ('https://matplotlib.org', None)
 }
 
-"""
-if os.environ.get('READTHEDOCS') == 'True':
+if os.environ.get('POLIASTRO_SKIP_NOTEBOOKS') == 'True':
     nbsphinx_execute = 'never'
 else:
     nbsphinx_execute = 'always'
 
+"""
```

Comment:
While we are at it, can we remove these lines of code that are commented out? If we want to bring them back, we will always have git :)

Reply:
Will remove them. 

I also don't understand when we do that. I think it's a habit we keep from the times when we didn't know how to use git.

Reply:
> I also don't understand when we do that. I think it's a habit we keep from the times when we didn't know how to use git.

Hahaha absolutely!

Reply:
> While we are at it, can we remove these lines of code that are commented out? If we want to bring them back, we will always have git :)

Done